crates/bevy_pbr/src/render/wireframe.wgsl

```diff
@@ -1,12 +1,12 @@
 #import bevy_pbr::forward_io::VertexOutput
 
-struct WireframeMaterial {
-    color: vec4<f32>,
-};
+struct PushConstants {
+    color: vec4<f32>
```

Comment:
FYI push constants aren't available on WebGPU. Idk how we feel about requiring them for wireframes.

Reply:
Wireframes require the `POLYGON_MODE_LINE` feature which isn't available on web.